### ISSUE TITLE
2.4_formats_and_performance - Adding chunking here?

### ISSUE THREAD

**Issue by annefou** [EO-College/cubes-and-clouds#52]
In this repo (not in EO-college yet), there is an episode [2.4 - Data Formats and Performance](https://github.com/EO-College/cubes-and-clouds/blob/main/lectures/2.4_formats_and_performance/2.4_formats_and_performance.md). I suggest to add the section about chunking here. 

What do you think?

**Comment by clausmichele** [comment]
Definitely, since that lecture is not yet published we can add a section about chunking, which is highly relevant.

**Comment by clausmichele** [comment]
@annefou @tinaok I guess that we should start from here, right? https://pangeo-data.github.io/geo-open-hack-2024/pangeo/chunking_introduction.html#what-is-a-chunk

**Comment by clausmichele** [comment]
To do: add image references
